detection_rules/main.py

```diff
@@ -180,25 +217,22 @@ def view_rule(ctx, rule_file, api_format):
                    '(an error will be raised otherwise)')
 def export_rules(rules, outfile, replace_id, stack_version, skip_unsupported):
     """Export rule(s) into an importable ndjson file."""
-    from .packaging import Package
-
     assert len(rules) > 0, "No rules found"
 
     if replace_id:
-        # if we need to replace the id, take each rule object and create a copy
-        # of it, with only the rule_id field changed
-        old_rules = rules
-        rules = RuleCollection()
+        from uuid import uuid4
+
+        new_rules = []
+        for rule in rules:
+            new_rules.append(rule.new(data=dict(rule_id=str(uuid4()))))
```

Comment:
put back if you can please

Reply:
Functionality was moved to `_export_rules` since it is shared. Can you elaborate otherwise?